### DISCUSSION TITLE
160mhz还是有问题

### DISCUSSION THREAD

**Discussion by ThermalEng** [Boos4721/openwrt#24] [Bugs/问题反馈]
之前有guestwifi的固件，ax200无论如何不能连接，且所有连接到5G的设备都严重掉速。
去掉guestwifi 重新编译后，所有5G设备速度正常。但ax200在160mhz仍不能连接。

**Comment by ThermalEng** [comment]
Tue Oct 12 07:22:03 2021 daemon.info hostapd: wlan1: STA 90:f0:52:55:41:b4 IEEE 802.11: authenticated
Tue Oct 12 07:22:03 2021 daemon.notice hostapd: wlan1: STA-OPMODE-N_SS-CHANGED 90:f0:52:55:41:b4 1
Tue Oct 12 07:22:03 2021 daemon.info hostapd: wlan1: STA 90:f0:52:55:41:b4 IEEE 802.11: associated (aid 2)
Tue Oct 12 07:22:03 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED 90:f0:52:55:41:b4
Tue Oct 12 07:22:03 2021 daemon.info hostapd: wlan1: STA 90:f0:52:55:41:b4 RADIUS: starting accounting session 137D0CF9DB6F8CB3
Tue Oct 12 07:22:03 2021 daemon.info hostapd: wlan1: STA 90:f0:52:55:41:b4 WPA: pairwise key handshake completed (RSN)
Tue Oct 12 07:22:03 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED 90:f0:52:55:41:b4

**Comment by ThermalEng** [comment]
之前连接失败的日志刷掉了，晚上回去再试试

**Comment by ThermalEng** [comment]
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712670] ath: EEPROM regdomain: 0x809c
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712682] ath: EEPROM indicates we should expect a country code
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712685] ath: doing EEPROM country->regdmn map search
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712688] ath: country maps to regdmn code: 0x52
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712692] ath: Country alpha2 being used: CN
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712694] ath: Regpair used: 0x52
Tue Oct 12 12:43:37 2021 kern.debug kernel: [  574.712697] ath: regdomain 0x809c dynamically updated by user
Tue Oct 12 12:44:42 2021 kern.info kernel: [  640.040285] device wlan1 left promiscuous mode
Tue Oct 12 12:44:42 2021 kern.info kernel: [  640.040476] br-lan: port 5(wlan1) entered disabled state
Tue Oct 12 12:44:42 2021 daemon.notice hostapd: Remove interface 'wlan1'
Tue Oct 12 12:44:42 2021 daemon.notice hostapd: wlan1: interface state ENABLED->DISABLED
Tue Oct 12 12:44:42 2021 daemon.notice hostapd: wlan1: AP-DISABLED
Tue Oct 12 12:44:42 2021 daemon.notice hostapd: wlan1: CTRL-EVENT-TERMINATING
Tue Oct 12 12:44:42 2021 daemon.notice hostapd: nl80211: deinit ifname=wlan1 disabled_11b_rates=0
Tue Oct 12 12:44:42 2021 daemon.notice hostapd: nl80211: Failed to remove interface wlan1 from bridge br-lan: Invalid argument
Tue Oct 12 12:44:42 2021 kern.info kernel: [  640.172104] wlan1: Destroyed NSS virtual interface
Tue Oct 12 12:44:42 2021 kern.info kernel: [  640.172139] wlan1: RX-Q purged.
Tue Oct 12 12:44:43 2021 kern.err kernel: [  640.508719] debugfs: File 'virt_if' in directory 'stats' already present!
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.509050] wlan1: Created a NSS virtual interface.
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.514579] wlan1: RX-Q and tasklet initialized.
Tue Oct 12 12:44:43 2021 daemon.notice hostapd: Configuration file: /var/run/hostapd-phy1.conf (phy wlan1) --> new PHY
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.561592] br-lan: port 5(wlan1) entered blocking state
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.561631] br-lan: port 5(wlan1) entered disabled state
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.566609] device wlan1 entered promiscuous mode
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.571604] br-lan: port 5(wlan1) entered blocking state
Tue Oct 12 12:44:43 2021 kern.info kernel: [  640.575905] br-lan: port 5(wlan1) entered forwarding state
Tue Oct 12 12:44:43 2021 daemon.notice hostapd: wlan1: interface state UNINITIALIZED->COUNTRY_UPDATE
Tue Oct 12 12:44:43 2021 daemon.notice hostapd: wlan1: interface state COUNTRY_UPDATE->HT_SCAN
Tue Oct 12 12:44:43 2021 daemon.notice hostapd: wlan1: interface state HT_SCAN->DFS
Tue Oct 12 12:44:43 2021 daemon.notice hostapd: wlan1: DFS-CAC-START freq=5220 chan=44 sec_chan=1, width=2, seg0=50, seg1=0, cac_time=1s
Tue Oct 12 12:44:43 2021 user.notice root: iptvhelper: reloading
Tue Oct 12 12:44:43 2021 user.notice root: iptvhelper: stopped
Tue Oct 12 12:44:43 2021 user.notice root: iptvhelper: starting
Tue Oct 12 12:44:43 2021 kern.info kernel: [  641.151925] br-lan: port 5(wlan1) entered disabled state
Tue Oct 12 12:44:54 2021 daemon.notice hostapd: wlan2: AP-STA-DISCONNECTED 94:17:00:40:38:5e
Tue Oct 12 12:45:00 2021 cron.err crond[10454]: USER root pid 15116 cmd /usr/bin/getnoddosdeviceprofiles;  if [ $? -gt 0 ]; then service noddos reload; fi
Tue Oct 12 12:45:43 2021 daemon.notice hostapd: wlan1: DFS-CAC-COMPLETED success=1 freq=5220 ht_enabled=0 chan_offset=0 chan_width=5 cf1=5250 cf2=0
Tue Oct 12 12:45:43 2021 daemon.warn hostapd: Can't set DFS state for freq 5180 MHz
Tue Oct 12 12:45:43 2021 daemon.warn hostapd: Can't set DFS state for freq 5200 MHz
Tue Oct 12 12:45:43 2021 daemon.warn hostapd: Can't set DFS state for freq 5220 MHz
Tue Oct 12 12:45:43 2021 daemon.warn hostapd: Can't set DFS state for freq 5240 MHz
Tue Oct 12 12:45:43 2021 kern.info kernel: [  701.442162] IPv6: ADDRCONF(NETDEV_CHANGE): wlan1: link becomes ready
Tue Oct 12 12:45:43 2021 kern.info kernel: [  701.442411] br-lan: port 5(wlan1) entered blocking state
Tue Oct 12 12:45:43 2021 kern.info kernel: [  701.447727] br-lan: port 5(wlan1) entered forwarding state
Tue Oct 12 12:45:43 2021 daemon.notice netifd: Network device 'wlan1' link is up
Tue Oct 12 12:45:43 2021 daemon.notice hostapd: wlan1: interface state DFS->ENABLED
Tue Oct 12 12:45:43 2021 daemon.notice hostapd: wlan1: AP-ENABLED
Tue Oct 12 12:46:51 2021 daemon.info hostapd: wlan1: STA 94:17:00:40:38:5e IEEE 802.11: authenticated
Tue Oct 12 12:46:51 2021 daemon.info hostapd: wlan1: STA 94:17:00:40:38:5e IEEE 802.11: associated (aid 1)
Tue Oct 12 12:46:51 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED 94:17:00:40:38:5e
Tue Oct 12 12:46:51 2021 daemon.info hostapd: wlan1: STA 94:17:00:40:38:5e RADIUS: starting accounting session D55F9F28257FF617
Tue Oct 12 12:46:51 2021 daemon.info hostapd: wlan1: STA 94:17:00:40:38:5e WPA: pairwise key handshake completed (RSN)
Tue Oct 12 12:46:51 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED 94:17:00:40:38:5e
Tue Oct 12 12:46:59 2021 daemon.notice hostapd: wlan2: AP-STA-DISCONNECTED 0c:dd:24:8f:6a:47
Tue Oct 12 12:46:59 2021 daemon.info hostapd: wlan1: STA 0c:dd:24:8f:6a:47 IEEE 802.11: authenticated
Tue Oct 12 12:46:59 2021 daemon.info hostapd: wlan1: STA 0c:dd:24:8f:6a:47 IEEE 802.11: associated (aid 2)
Tue Oct 12 12:47:08 2021 daemon.info hostapd: wlan1: STA 0c:dd:24:8f:6a:47 IEEE 802.11: deauthenticated due to local deauth request
Tue Oct 12 12:47:12 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED 94:17:00:40:38:5e
Tue Oct 12 12:47:12 2021 daemon.info hostapd: wlan2: STA 94:17:00:40:38:5e IEEE 802.11: authenticated
Tue Oct 12 12:47:13 2021 daemon.info hostapd: wlan2: STA 94:17:00:40:38:5e IEEE 802.11: authenticated
Tue Oct 12 12:47:13 2021 daemon.info hostapd: wlan2: STA 94:17:00:40:38:5e IEEE 802.11: associated (aid 1)
Tue Oct 12 12:47:13 2021 daemon.notice hostapd: wlan2: AP-STA-CONNECTED 94:17:00:40:38:5e
Tue Oct 12 12:47:13 2021 daemon.info hostapd: wlan2: STA 94:17:00:40:38:5e RADIUS: starting accounting session 3D3E175B568082F0
Tue Oct 12 12:47:13 2021 daemon.info hostapd: wlan2: STA 94:17:00:40:38:5e WPA: pairwise key handshake completed (RSN)
Tue Oct 12 12:47:13 2021 daemon.notice hostapd: wlan2: EAPOL-4WAY-HS-COMPLETED 94:17:00:40:38:5e
Tue Oct 12 12:47:15 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 IEEE 802.11: associated (aid 2)
Tue Oct 12 12:47:15 2021 daemon.notice hostapd: wlan2: AP-STA-CONNECTED 0c:dd:24:8f:6a:47
Tue Oct 12 12:47:15 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 RADIUS: starting accounting session 516889F707F06305
Tue Oct 12 12:47:15 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 WPA: pairwise key handshake completed (RSN)
Tue Oct 12 12:47:15 2021 daemon.notice hostapd: wlan2: EAPOL-4WAY-HS-COMPLETED 0c:dd:24:8f:6a:47
设为不加密网络仍然连不上：
Tue Oct 12 22:12:46 2021 kern.info kernel: [  616.642447] device wlan1 left promiscuous mode
Tue Oct 12 22:12:46 2021 kern.info kernel: [  616.642622] br-lan: port 5(wlan1) entered disabled state
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: Remove interface 'wlan1'
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: wlan1: interface state ENABLED->DISABLED
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED 94:17:00:40:38:5e
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED 98:e0:d9:cf:0b:0e
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: wlan1: AP-DISABLED
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: wlan1: CTRL-EVENT-TERMINATING
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: nl80211: deinit ifname=wlan1 disabled_11b_rates=0
Tue Oct 12 22:12:46 2021 daemon.notice hostapd: nl80211: Failed to remove interface wlan1 from bridge br-lan: Invalid argument
Tue Oct 12 22:12:51 2021 kern.warn kernel: [  621.730161] ath11k c000000.wifi: failed to flush transmit queue 0
Tue Oct 12 22:12:51 2021 kern.info kernel: [  621.811725] wlan1: Destroyed NSS virtual interface
Tue Oct 12 22:12:51 2021 kern.info kernel: [  621.811760] wlan1: RX-Q purged.
Tue Oct 12 22:12:52 2021 user.notice root: iptvhelper: reloading
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020283] ath: EEPROM regdomain: 0x809c
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020295] ath: EEPROM indicates we should expect a country code
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020298] ath: doing EEPROM country->regdmn map search
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020301] ath: country maps to regdmn code: 0x52
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020305] ath: Country alpha2 being used: CN
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020308] ath: Regpair used: 0x52
Tue Oct 12 22:12:52 2021 kern.debug kernel: [  622.020311] ath: regdomain 0x809c dynamically updated by user
Tue Oct 12 22:12:52 2021 user.notice root: iptvhelper: stopped
Tue Oct 12 22:12:52 2021 user.notice root: iptvhelper: starting
Tue Oct 12 22:12:53 2021 kern.err kernel: [  623.168653] debugfs: File 'virt_if' in directory 'stats' already present!
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.170052] wlan1: Created a NSS virtual interface.
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.174539] wlan1: RX-Q and tasklet initialized.
Tue Oct 12 22:12:53 2021 daemon.notice hostapd: Configuration file: /var/run/hostapd-phy1.conf (phy wlan1) --> new PHY
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.219261] br-lan: port 5(wlan1) entered blocking state
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.219295] br-lan: port 5(wlan1) entered disabled state
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.224212] device wlan1 entered promiscuous mode
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.229272] br-lan: port 5(wlan1) entered blocking state
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.233686] br-lan: port 5(wlan1) entered listening state
Tue Oct 12 22:12:53 2021 daemon.notice hostapd: wlan1: interface state UNINITIALIZED->COUNTRY_UPDATE
Tue Oct 12 22:12:53 2021 daemon.notice hostapd: wlan1: interface state COUNTRY_UPDATE->HT_SCAN
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.239871] br-lan: port 5(wlan1) entered disabled state
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.584881] IPv6: ADDRCONF(NETDEV_CHANGE): wlan1: link becomes ready
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.585122] br-lan: port 5(wlan1) entered blocking state
Tue Oct 12 22:12:53 2021 kern.info kernel: [  623.590353] br-lan: port 5(wlan1) entered listening state
Tue Oct 12 22:12:53 2021 daemon.notice hostapd: wlan1: interface state HT_SCAN->ENABLED
Tue Oct 12 22:12:53 2021 daemon.notice hostapd: wlan1: AP-ENABLED
Tue Oct 12 22:12:54 2021 daemon.notice netifd: Network device 'wlan1' link is up
Tue Oct 12 22:13:01 2021 kern.info kernel: [  631.714836] br-lan: port 5(wlan1) entered learning state
Tue Oct 12 22:13:02 2021 daemon.notice hostapd: wlan2: AP-STA-DISCONNECTED 0c:dd:24:8f:6a:47
Tue Oct 12 22:13:10 2021 kern.info kernel: [  639.907378] br-lan: port 5(wlan1) entered forwarding state
Tue Oct 12 22:13:10 2021 kern.info kernel: [  639.907416] br-lan: topology change detected, propagating
Tue Oct 12 22:13:11 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 IEEE 802.11: authenticated
Tue Oct 12 22:13:11 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 IEEE 802.11: associated (aid 5)
Tue Oct 12 22:13:11 2021 daemon.notice hostapd: wlan2: AP-STA-CONNECTED 0c:dd:24:8f:6a:47
Tue Oct 12 22:13:11 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 RADIUS: starting accounting session CCA75DA737262C44
Tue Oct 12 22:13:11 2021 daemon.info hostapd: wlan2: STA 0c:dd:24:8f:6a:47 WPA: pairwise key handshake completed (RSN)
Tue Oct 12 22:13:11 2021 daemon.notice hostapd: wlan2: EAPOL-4WAY-HS-COMPLETED 0c:dd:24:8f:6a:47

**Comment by ThermalEng** [comment]
很奇怪，在没有修改ssid和密码的情况下，连接都没有问题，一旦一修改默认配置，马上就变得不再正常。

**Comment by ThermalEng** [comment]
成功定位问题，不能关闭WMM。一旦关闭， 即出现如上情况。打开的情况下wifi正常。

**Comment by ThermalEng** [comment]
另外提一个小问题，adh的可执行文件路径是 /usr/bin/AdGuardHome/AdGuardHome，而非 /usr/bin/AdGuardHome

**Reply by Boos4721** [reply]
> 另外提一个小问题，adh的可执行文件路径是 /usr/bin/AdGuardHome/AdGuardHome，而非 /usr/bin/AdGuardHome

https://github.com/Boos4721/OpenWrt-Packages/blob/71d1372b49e8ec480214dd9afcdd78daa635f062/AdGuardHome/Makefile#L91
